packages/cli/generators/relation/templates/controller-relation-template-belongs-to.ts.ejs

```diff
@@ -24,7 +24,7 @@ export class <%= controllerClassName %> {
         description: '<%= targetModelClassName %> belonging to <%= sourceModelClassName %>',
         content: {
           'application/json': {
-            schema: {type: 'array', items: getModelSchemaRef(<%= targetModelClassName %>)},
+            schema: {type: 'object', items: getModelSchemaRef(<%= targetModelClassName %>)},
```

Comment:
```suggestion
            schema: getModelSchemaRef(<%= targetModelClassName %>),
```

Reply:
Don't we need to have the type added here?

Reply:
@samarpan-b, WDYT?

Reply:
Looking at the hasOne relation [template](https://github.com/loopbackio/loopback-next/blob/febc3bad5be348789ce11ebb2ad22c2d3199d0cb/packages/cli/generators/relation/templates/controller-relation-template-has-one.ts.ejs#L35), it doesn't have `type` under the `schema`. So this isn't probably required but in case of a hasOne relation or belongsTo relation, the returning data would be an object all the time. Isn't it the case? 
@dhmlau , @samarpan-b

Reply:
type object is not needed. items work with type array only. If its an object, then we can directly refer that model schema. Thats what I suggested. You can test this as well.

Reply:
I've just tested it. We really don't need a type here. But then do we need to type in the specs at all? I mean it is present in the belongsTo, hasMany & hasManyThrough relations but not in the hasOne relation.

Should it be removed from other relations too?

Reply:
for cases where there is an array in response, its needed. I think its okay in other places.

Reply:
I've updated the PR. Please have a look now and let me know if we can get it merged. @samarpan-b